frontend/src/utils/api.js

```diff
@@ -183,6 +183,7 @@ window.api = {
     window.repoSenseVersion = data.repoSenseVersion;
     window.isSinceDateProvided = data.isSinceDateProvided;
     window.isUntilDateProvided = data.isUntilDateProvided;
+    document.title = data.reportTitle || 'RepoSense Report';
```

Comment:
I believe `document.title = data.reportTitle || document.title;` would be better, as I prefer only a single source for the tab default name. I think the default name can be changed in `package.json`.

Reply:
I mean `index.html`, my bad.

Reply:
Thanks for the suggestion! I was looking for a way to standardize the default name with backend but couldn't think of any.